nilearn/plotting/img_plotting.py

```diff
@@ -815,6 +834,32 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
     return display
 
 
+def _cmap_discretize(cmap, N):
+    """Return a discrete colormap from the continuous colormap cmap.
+
+        cmap: colormap instance, eg. cm.jet.
+        N: number of colors.
+
+    Example
+        x = resize(arange(100), (5,100))
+        djet = cmap_discretize(cm.jet, 5)
+        imshow(x, cmap=djet)
```

Comment:
This is unused anyway
Le 26 févr. 2016 17:33, "Gael Varoquaux" <notifications@github.com> a
écrit :

> In nilearn/plotting/img_plotting.py
> <https://github.com/nilearn/nilearn/pull/1026#discussion_r54266010>:
>
> > @@ -815,6 +834,32 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
> >      return display
> >
> >
> > +def _cmap_discretize(cmap, N):
> > +    """Return a discrete colormap from the continuous colormap cmap.
> > +
> > +        cmap: colormap instance, eg. cm.jet.
> > +        N: number of colors.
> > +
> > +    Example
> > +        x = resize(arange(100), (5,100))
> > +        djet = cmap_discretize(cm.jet, 5)
> > +        imshow(x, cmap=djet)
>
> Could you format this according to the numpy doc standard: a Parameters
> section, and Example as a section.
>
> The reason that I ask this, is that the docstring is invalid currently.
>
> Also, the examples should be doctests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/1026/files#r54266010>.
>